uni/lib/view/Pages/useful_contacts_card_page_view.dart

```diff
@@ -36,15 +36,15 @@ Container infoText(String text, BuildContext context,
         child: InkWell(
             child: Text(
               text,
-              style: Theme.of(context).textTheme.bodyText1.apply(
-                    color: Theme.of(context).accentColor,
-                  ),
+              style: Theme.of(context).textTheme.bodyText1!.apply(),
```

Comment:
There is a technical reason to not apply the accent color? 
The page lost contrast when there is no red (for light theme), I think

Reply:
The accent color is no more, however we can surely apply the tertiary color manually for details like these.